src/utils/flashMintNotional/fmNotionalContract.ts

```diff
@@ -0,0 +1,20 @@
+import { Provider } from '@ethersproject/abstract-provider'
+import { Signer } from '@ethersproject/abstract-signer'
+import { Contract } from '@ethersproject/contracts'
+
+import { FLASH_MINT_NOTIONAL_ABI } from './FlashMintNotionalAbi'
+
+export const FlashMintNotionalContractAddress =
+  '0x9DA9992b5d01BD0EFb1EE8310E8011dc837bd476'
+
+export const getFlashMintNotionalContract = (
+  providerOrSigner: Signer | Provider | undefined,
+  chainId: number = 1
+): Contract => {
+  const contractAddress = FlashMintNotionalContractAddress
+  return new Contract(
```

Comment:
Have you tried / thought about just importing the typechain artifacts / factories from the smart contract repo instead of copy pasting the abi ?

Reply:
As discussed, will improve once moving to the code to the FlashMintSDK.